poly-commit/src/hyrax/mod.rs

```diff
@@ -272,15 +272,16 @@ impl<G: AffineRepr, P: MultilinearExtension<G::ScalarField>>
             let m = flat_to_matrix_column_major(&poly.to_evaluations(), dim, dim);
 
             // Commiting to the matrix with one multi-commitment per row
-            let row_coms = m
-                .iter()
+            let (row_coms, com_rands): (Vec<_>, Vec<_>) = cfg_iter!(m)
                 .map(|row| {
+                    #[cfg(not(feature = "parallel"))]
                     let (c, r) = Self::pedersen_commit(ck, row, None, Some(rng_inner));
-                    // Storing the randomness used in the commitment
-                    com_rands.push(r);
-                    c
+                    #[cfg(feature = "parallel")]
+                    let (c, r) =
+                        Self::pedersen_commit(ck, row, None, Some(&mut rand::thread_rng()));
```

Comment:
Is this `thread_rng` strong enough? (e.g. stronger than the `test_rng`s we normally use for testing). If so, I'm happy with all the code

Reply:
It's definitely stronger than test_rng :D

Reply:
Seems solid: https://rust-random.github.io/rand/rand/fn.thread_rng.html

Merge away!